docs/user/integration/dynatrace/README.md

```diff
@@ -0,0 +1,154 @@
+
+### Create Secret
+
+1. To create a new secret containing your access token. In the following command, replace the `{API_TOKEN}` placeholder with the previously created token and run it::
+    ```bash
+    kubectl -n $DYNATRACE_NS create secret generic dynatrace-token --from-literal="apiToken=Api-Token {API_TOKEN}"
+    ```
+
+### Ingest Traces
+
+#### Enable Collecting Istio Traces
+
+By default, the tracing feature of the Istio service mesh is disabled to save resources for the case that no TracePipeline is defined.  To enable emitting traces with the kyma-traces Istio extension provider, create a Telemetry custom resource in the `istio-system` Namespace:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: tracing-default
+  namespace: istio-system
+spec:
+  tracing:
+  - providers:
+    - name: "kyma-traces"
+    randomSamplingPercentage: 1.0
+EOF
+```
+
+This configuration samples 1% of all requests. if you want to change the sampling rate, adjust the **randomSamplingPercentage** property.
+
+#### Create Kyma TracePipeline
+
+1. Replace the `{ENVIRONMENT_ID}` placeholder with the environment Id of your Dynatrace instance:
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: telemetry.kyma-project.io/v1alpha1
+    kind: TracePipeline
+    metadata:
+        name: dynatrace
+    spec:
+        output:
+            otlp:
+                endpoint:
+                    value: https://apm.cf.eu12.hana.ondemand.com/e/{ENVIRONMENT_ID}/api/v2/otlp
+                headers:
+                    - name: Authorization
+                      valueFrom:
+                          secretKeyRef:
+                              name: dynatrace-token
+                              namespace: ${DYNATRACE_NS}
+                              key: apiToken
+                protocol: http
+    EOF
+    ```
+1. To find traces from your Kyma cluster in the Dynatrace UI, go to **Applications & Microservices** > **Distributed traces**.
+
+### Ingest Metrics
+
+#### Create Kyma MetricPipeline
+
+1. Replace the `{ENVIRONMENT_ID}` placeholder with the environment Id of your Dynatrace instance:
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: telemetry.kyma-project.io/v1alpha1
+    kind: MetricPipeline
+    metadata:
+        name: dynatrace
+    spec:
+        output:
+            convertToDelta: true
+            otlp:
+                endpoint:
+                    value: https://apm.cf.eu12.hana.ondemand.com/e/{ENVIRONMENT_ID}/api/v2/otlp
+                headers:
+                    - name: Authorization
+                      valueFrom:
+                          secretKeyRef:
+                              name: dynatrace-token
+                              namespace: ${DYNATRACE_NS}
+                              key: apiToken
+                protocol: http
+    EOF
+    ```
+
+## Verify the results by deploying sample apps
+
```

Comment:
It should be enough to reference the existing tutorial for that, isn't it?

What is with the dynatrace operator, don't we need to install it as well?

How to rverify metrics are arriving? You describe it for traces but not for metrics